examples/mpi/lammps/Dockerfile

```diff
@@ -1,29 +1,41 @@
 # ch-test-scope: full
 FROM openmpi
-WORKDIR /
+WORKDIR /usr/local/src
 
 # Packages for building.
 RUN apt-get install -qy --no-install-recommends \
+    cmake \
     git \
     patch \
-    python-dev
+    python-dev \
+    python-pip \
+    python-setuptools 
 
+# Building mpi4py from source to ensure it is built against our MPI build
+# Building numpy from source to work around issues seen on Aarch64 systems
+RUN pip install --no-binary=mpi4py,numpy mpi4py numpy 
 # Build LAMMPS.
-ENV LAMMPS_VERSION 17Nov16
-ENV LAMMPS_DIR lammps-$LAMMPS_VERSION
-ENV LAMMPS_TAR $LAMMPS_DIR.tar.gz
-RUN wget -nv http://lammps.sandia.gov/tars/$LAMMPS_TAR
-RUN tar xf $LAMMPS_TAR
-RUN    cd $LAMMPS_DIR/src \
-    && python Make.py -j $(getconf _NPROCESSORS_ONLN) -p none \
-              std no-lib reax meam poems python reaxc orig -a lib-all mpi
-RUN    mv $LAMMPS_DIR/src/lmp_mpi /usr/bin \
-    && ln -s ./$LAMMPS_DIR /lammps
-RUN    cd $LAMMPS_DIR/python \
-    && python2.7 install.py
-
-# Make another test input file.
-# Patch instead of including in.large.melt in full because LAMMPS is GPL.
-COPY melt.patch /lammps/examples/melt
-RUN patch -p1 -o /lammps/examples/melt/in.large.melt \
-               < /lammps/examples/melt/melt.patch
+ENV LAMMPS_VERSION 19Sep19
+RUN wget -nv http://lammps.sandia.gov/tars/lammps-$LAMMPS_VERSION.tar.gz
+RUN tar xf lammps-$LAMMPS_VERSION.tar.gz
+RUN mkdir lammps-${LAMMPS_VERSION}.build
+RUN    cd lammps-${LAMMPS_VERSION}.build \
+    && cmake -DCMAKE_INSTALL_PREFIX=/usr \
```

Comment:
Inconsistent with `/usr/local/src` above.

Reply:
Good catch, I'll change to `&& cmake -DCMAKE_INSTALL_PREFIX=/usr/local \`